apps/front-end/src/serviceWorkerRegistration.js

```
@@ -34,25 +34,25 @@ export function register(config) {
         installingWorker.onstatechange = () => {
-          if (installingWorker.state === 'installed') {
+          if (installingWorker.state === "installed") {
             if (navigator.serviceWorker.controller) {
               console.log(
-                'New content is available and will be used when all ' +
-                'tabs for this page are closed. See https://cra.link/PWA.'
+                "New content is available and will be used when all " +
+                  "tabs for this page are closed. See https://cra.link/PWA.",
               );
 
               if (config && config.onUpdate) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)